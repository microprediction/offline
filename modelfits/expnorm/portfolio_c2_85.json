{"g1": 0.04652143542323074, "g2": 1.4962678391170499, "logK": -2.759283733692478, "loc": 0.03405641683531798, "logScale": 1.0311417102038931}