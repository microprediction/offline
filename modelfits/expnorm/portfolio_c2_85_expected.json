{"g1": 0.2727641839052637, "g2": 14.988116255910708, "logK": 0.9576623832226758, "loc": -0.0865067193209475, "logScale": -2.2423263977591352}